<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelForm</class>
 <widget class="QWidget" name="DelForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btn_add">
       <property name="text">
        <string>增加</string>
       </property>
       <property name="icon">
        <iconset>
         <activeon>:/icon/add.png</activeon>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_del">
       <property name="text">
        <string>删除</string>
       </property>
       <property name="icon">
        <iconset>
         <activeon>:/icon/del.png</activeon>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_edit">
       <property name="text">
        <string>修改</string>
       </property>
       <property name="icon">
        <iconset>
         <activeon>:/icon/edit.png</activeon>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_show">
       <property name="text">
        <string>保存</string>
       </property>
       <property name="icon">
        <iconset>
         <activeon>:/icon/save .png</activeon>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
